Sources/Utility/Process.swift

```diff
@@ -165,6 +172,24 @@ public final class Process: ObjectIdentifierProtocol {
         self.redirectOutput = redirectOutput 
     }
 
+    /// Validate that the program provided exists on the search path and is executable.
+    internal func validateArgumentIsExecutable(_ program: String) throws {
```

Comment:
oh I meant that the default scope is internal and we don't explicitly write internal in swiftpm's codebase.
For e.g., these two structs have similar scope:
```swift
struct Foo {}
```
```swift
internal struct Foo{}
```